### DISCUSSION TITLE
How to gather test results for entry into a database

### DISCUSSION THREAD

**Discussion by cemerick1** [getgauge/gauge#2435] [General]
I was curious to know if there was a way to gather test results so as to enter them into the database effeciently.  Currently we use a combination of the json report and the xml report to get the data but its quite a pain to do it that way.  How do these plugins get the data? Is there a method being overwritten? Can I just gather this data using an already provided method by Gauge? Thanks for any help! God Bless!

**Comment by zabil** [comment]
You can refer https://github.com/getgauge-contrib/report-seed understand how data is passed after an execution. Possibly use that as a base to write your own plugin.
